linux/runtime.go

```diff
@@ -147,7 +157,36 @@ func (r *Runtime) Create(ctx context.Context, id string, opts runtime.CreateOpts
 			bundle.Delete()
 		}
 	}()
-	s, err := bundle.NewShim(ctx, r.shim, r.address, r.remote, r.shimDebug, opts)
+	s, err := bundle.NewShim(ctx, r.shim, r.address, r.remote, r.shimDebug, opts, func() {
+		t, err := r.tasks.Get(ctx, id)
+		if err != nil {
+			// Task was never started or was already sucessfully deleted
+			return
+		}
+		lc := t.(*Task)
```

Comment:
any chance this is never `!ok`? I'm not aware enough of the `Task` type to understand the reference assertion here.

Reply:
The runtime only store that type in the list, so it should always be `ok`. If it's not, happy to see a panic so we can fix it :)